src/runtime/composables/turnstile.ts

```diff
@@ -0,0 +1,19 @@
+import { useScript } from "#imports"
+
+declare global {
+    interface Window {
+        turnstile: Turnstile.Turnstile
+    }
+}
+
+export function useScriptCloudflareTurnstile() {
+    return useScript({
+        src: 'https://challenges.cloudflare.com/turnstile/v0/api.js',
+        async: true,
+        defer: true,
+    }, {
+        // do not bundle. Turnstile must have it's own script tag.
```

Comment:
What happens if you bundle it 🤔 ?

Reply:
turnstile is looking for the turnstile script when loading so we cannot bundle it